backend/tests/test_server.py

```diff
@@ -25,24 +27,36 @@ def __init__(self) -> None:
                 value = "=".join(split_line[1:]).strip("\n")
                 os.environ[name] = value
         
-
-
         self.db_client, self.cdl_db = self.get_db()
-        self.URL = os.environ["api_url"] + ":" + os.environ["api_port"] 
-
-
+        self.URL = os.environ["api_url"] + ":" + os.environ["api_port"]
 
     def get_db(self):
         client = MongoClient([os.environ["cdl_test_uri"]])
         cdl_db = client[os.environ["db_name"]]
         return client, cdl_db
 
     def clear_db(self):
-        if self.db_client.address == ('localhost', 27017):
+        if self.db_client.address == ('0.0.0.0', 27017):
```

Comment:
0.0.0.0 is required as its the address we bind mongodb to be able to access it from outside the container.